dogen/templates/template.jinja

```diff
@@ -130,6 +127,17 @@ WORKDIR {{ workdir }}
 
 USER {{ user }}
 
+{% if cct %}
+# Add cct in command mode as entrypoint
+# (this overrides any user-supplied entrypoint)
+# you can specify path to changes files in CCT_CHANGES environment variable
+ENTRYPOINT ["/usr/bin/cct", {% if cct.runtime_changes -%}"{{ cct.runtime_changes }}", {% endif -%}"-c"]
+{% else %}
+{% if entrypoint %}
+ENTRYPOINT {{ helper.cmd(entrypoint) }}
+{% endif %}
```

Comment:
it cct plugin is used it will use old entrypoint not a new one, right? this seems that you set cfg['entrypoint'] in cct plugin but dont use it in template.